scripts/build.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs-extra');
+const path = require('path');
+const { getSrcFilePath, buildTemplateParameters, buildCode } = require('../src/helpers');
+const { main: mainPath } = require('../package.json');
+
+const DIST_PATH = path.resolve(mainPath);
+
+async function build() {
+  const testsContent = await fs.readFile(getSrcFilePath('tests.txt'));
```

Comment:
@InakiAbete-Reflektion you can use `readFileSync` instead if you want to do this sequentially.

Reply:
AFAIK, this way is a non-blocking way to do it sequentially, `readFileSync` blocks the main thread.